src/components/grapher/Grapher.vue

```diff
@@ -3,12 +3,21 @@
     <svg class="grapher-svg"><g> <!-- Note: we wrap the entire svg in a g so that svgPanZoom doesn't
             have to create a new g and Vue loses track of the DOM elements. -->
       <!-- Note:Inside svg, 1px = 1 eight-to-five step -->
-      <g class="grapher--line-container">
+      <g
+        class="grapher--field-container"
```

Comment:
Is this class needed?

Reply:
No, you're right! artifact from the field rectangle checkbox...